scalding-core/src/main/scala/com/twitter/scalding/serialization/KryoHadoop.scala

```diff
@@ -61,7 +64,7 @@ class KryoHadoop(config: Config) extends KryoInstantiator {
     newK.register(classOf[DateRange], new DateRangeSerializer())
     newK.register(classOf[Args], new ArgsSerializer)
     // Some of the monoids from Algebird that we use:
-    newK.register(classOf[com.twitter.algebird.AveragedValue], new AveragedValueSerializer)
+    newK.register(classOf[com.twitter.algebird.AveragedValue], new  AveragedValueSerializer)
```

Comment:
I'll commit it, but see in line 64, where before it said: "new AveragedValuSerializer" now it has two spaces: "new  AveragedValueSerializer".

The larger point is that your diff should be as minimal as possible when making a functional change (such as this one). Style changes with zero functional changes can be larger (as review is simpler: it is only making sure all the changes are stylistic).